Modules/Server/Sources/PocketCastsServer/Public/Refresh/MainServerHandler.swift

```diff
@@ -353,6 +353,73 @@ public class MainServerHandler {
         }.resume()
     }
 
+    public func updatePodcast(uuid: String, lastEpisodeUuid: String?) async throws -> Bool {
+        var query = "podcast_uuid=\(uuid)"
+        if let lastEpisodeUuid {
+            query += "&last_episode_uuid=\(lastEpisodeUuid)"
+        }
+        let url = ServerHelper.asUrl(ServerConstants.Urls.main() + "api/v1/update_podcast?\(query)")
+        var request = URLRequest(url: url)
+        request.httpMethod = "GET"
+        request.cachePolicy = .reloadIgnoringLocalCacheData
+
+        FileLog.shared.console("Update Podcast API start request \(url.absoluteString)")
+
+        if Task.isCancelled {
+            return false
+        }
+
+        let response = try await URLSession.shared.data(for: request)
+        guard let urlResponse = response.1 as? HTTPURLResponse else {
+            return false
+        }
+
+        FileLog.shared.console("Update Podcast API response status code \(urlResponse.statusCode)")
+
+        var statusCode = urlResponse.statusCode
+        let allHeaderFields = urlResponse.allHeaderFields
+        while statusCode == 202 {
```

Comment:
Uses the same logic as Android in [here](https://github.com/Automattic/pocket-casts-android/commit/4696fa6fee81ec8ae24d9651f3127606098e9bed)